lib/shared/errors.js

```diff
@@ -228,7 +234,7 @@ exports.getDescription = (error) => {
   }
 
   if (error.stack) {
```

Comment:
Or you could just add a more detailed inline code-comment, explaining why we return an empty string if `error.stack` is set (which from what you say above, indicates it's a "standard" Error?) and `considerStacktrace` is false.  ;-)

Or perhaps `userFriendlyDescriptionsOnly` is a better name for `ignoreNonUserFriendlyForms` ?